ratis-proto/src/main/proto/Raft.proto

```diff
@@ -148,6 +148,7 @@ enum InstallSnapshotResult {
   IN_PROGRESS = 2;
   ALREADY_INSTALLED = 3;
   CONF_MISMATCH = 4;
+  NULL_SNAPSHOT = 5;
```

Comment:
How about calling it as SNAPSHOT_UNAVALIABLE?

Reply:
Sure. Updated.